packages/apps/storybook/src/utils/storyHelp.ts

```diff
@@ -20,13 +20,10 @@ export const withAccessTokenOverride: <
 ) => Story<T> = (Story) => (args, context) =>
   Story({ ...args, accessToken: context.globals.accessToken }, context);
 
-/** HOC that will override the "projectId" prop with the Addon projectId */
+/** HOC that will override the "iTwinId" prop with the Addon iTwinId */
 export const withProjectIdOverride: <
-  T extends { projectId?: string | undefined }
+  T extends { iTwinId?: string | undefined }
 >(
   story: Story<T>
 ) => Story<T> = (Story) => (args, context) =>
-  Story(
-    { ...args, projectId: args.projectId || context.globals.projectId },
-    context
-  );
+  Story({ ...args, iTwinId: args.iTwinId || context.globals.iTwinId }, context);
```

Comment:
If you change the `globals.projectId` here, you will need to update the addon, or keep `projectId`.

This storybook global is defined in here: `/packages/apps/storybook/src/addon/register.js`

Also, since we are changing this, you might as well change the name of the HOC to `withITwinIdOverride`

Reply:
Done